api/v1/submissions/PKPSubmissionHandler.inc.php

```diff
@@ -59,6 +59,12 @@ class PKPSubmissionHandler extends APIHandler
         'publishPublication',
         'unpublishPublication',
         'deletePublication',
+        'getContributors',
+        'getContributor',
+        'addContributor',
+        'deleteContributor',
+        'editContributor',
+        'saveContributorsOrder',
```

Comment:
The following should be added to the `$requiresPublicationWriteAccess` array as well:

```
      'addContributor',
        'deleteContributor',
        'editContributor',
        'saveContributorsOrder',
```

Reply:
Fixed